Makefile

```diff
@@ -1,15 +1,26 @@
 BIN_OUTPUT_PATH = bin
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
+ARM64_OUTPUT = $(BIN_OUTPUT_PATH)/raspberry-pi/arm64
+ARM32_OUTPUT = $(BIN_OUTPUT_PATH)/raspberry-pi/arm32
```

Comment:
These no longer match how the module itself builds. I'd revert and keep the single, old location (more details on that below.)